media/server/gstplayer/source/GstGenericPlayer.cpp

```diff
@@ -908,6 +912,58 @@ void GstGenericPlayer::updatePlaybackGroup(GstElement *typefind, const GstCaps *
     m_workerThread->enqueueTask(m_taskFactory->createUpdatePlaybackGroup(m_context, typefind, caps));
 }
 
+void GstGenericPlayer::addAutoVideoSinkChild(GObject *object)
+{
+    // Only add children that are sinks
+    if (GST_OBJECT_FLAG_IS_SET(GST_ELEMENT(object), GST_ELEMENT_FLAG_SINK))
+    {
+        RIALTO_SERVER_LOG_DEBUG("Store AutoVideoSink child sink");
+
+        if (m_context.autoVideoChildSink && m_context.autoVideoChildSink != GST_ELEMENT(object))
+        {
+            RIALTO_SERVER_LOG_WARN("AutoVideoSink child is been overwritten");
```

Comment:
Why is it warning? Wouldn't milestone be better?

Reply:
done